pyuvdata/uvdata/tests/test_mwa_corr_fits.py

```diff
@@ -644,6 +635,7 @@ def test_remove_coarse_band():
     uv2.data_array = uv2.data_array.swapaxes(2, 3)
     uv2.data_array = uv2.data_array / cb_array
     uv2.data_array = uv2.data_array.swapaxes(2, 3)
+    uv2.history = uv1.history
```

Comment:
Same comment here about checking the history before replacing it.

Reply:
fixed